docs/contribute/benchmarks/TRAINING/benchmarks_DL2_direction-reconstruction.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>we might want to look into later why this is different (not sure anyone has ever really tested the shower-max reco), but since the reconstructed event selection is also different, it's not too bad. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/protopipe/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>